tests/playwright/sanity/modules/playing-cards/playing-cards.test.ts

```diff
@@ -0,0 +1,26 @@
+import { expect, type Page, type BrowserContext } from '@playwright/test';
+import { parallelTest as test } from '../../../parallelTest';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorPage from '../../../pages/editor-page';
+
+test.describe( 'Playing cards widget sanity test', () => {
+	let editor: EditorPage;
+	let wpAdmin: WpAdminPage;
+	let context: BrowserContext;
+	let page: Page;
+
+	test.beforeEach( async ( { browser, apiRequests }, testInfo ) => {
+		context = await browser.newContext();
+		page = await context.newPage();
+		wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+		await wpAdmin.resetExperiments();
+		editor = await wpAdmin.openNewPage();
+	} );
+
+	test( 'it successfully adds card', async () => {
```

Comment:
@max-zu You need to upload the image that was create on the CI run (you can download the artifacts), not one from your localhost.

Reply:
Image name should end with `-linux` not `-darwin` :)